tests/stub/routing.py

```diff
@@ -219,7 +270,7 @@ def router_script_with_one_reader_and_exit(self):
 
         C: HELLO {"scheme": "basic", "credentials": "c", "principal": "p", "user_agent": "007", "routing": #HELLO_ROUTINGCTX# #EXTRA_HELLO_PROPS#}
         S: SUCCESS {"server": "#SERVER_AGENT#", "connection_id": "bolt-123456789"}
-        C: ROUTE #ROUTINGCTX# [] "adb"
+        C: ROUTE #ROUTINGCTX# [] "*"
```

Comment:
Was there some need to support any database?

Reply:
`test_should_use_resolver_during_rediscovery_when_existing_routers_fail` uses this router for a session on `adb` while `test_should_successfully_acquire_rt_when_router_ip_changes` uses it to `verifyConnectivity`. Side note: I'm not a fan of (ab)using `verifyConnectivity` to force the driver to fetch a new RT because a) I'm not sure we ever specified that function to do that, b) at least the JS driver does something different, and c) the method's name has nothing to do with routing tables, why should one assume it always and only updates the RT?. I think in the long run, we should have another method exposed by the testkit backend to do this.

Reply:
This is simply the legacy of java driver. When those tests were introduced there, they probably did not expect it to be moved to testkit one day... It can and should be improved later on :)